api/viewsets.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import viewsets
```

Comment:
the name of the entirefile should be api/views.py to keep the standard django conventions. No matter that the only thing around is viewsets ;)